GEOSwift.podspec

```diff
@@ -18,6 +18,6 @@ DESC
   s.source       = { :git => "https://github.com/GEOSwift/GEOSwift.git", :tag => s.version }
 
   s.source_files = "GEOSwift/*.{swift,h}"
-  s.dependency "geos", "3.5.0"
+  s.dependency "geos"
```

Comment:
I would prefer you explicitly state the version here, in this case 3.5.1.  The "geos" 3.5.0 is using a svn repo, which is failing to checkout for me as well.

Reply:
then we have to set it to 3.6.2. I'm waiting for @vfn to give me access to `geos podspec` so I can update both. 
It's already updated here.
`source 'https://github.com/mojtabacazi/Specs.git'`

Reply:
@mojtabacazi please let me know what email you would like to use for accessing the podspecs

Reply:
@vfn same github account

Reply:
@mojtabacazi I can't see your email. It's probably set to private on your settings. https://github.com/settings/emails